### ISSUE TITLE
Suport custom model in text recognition

### ISSUE THREAD

**Issue by b4ra0** [flutter-ml/google_ml_kit_flutter#464]
I use a specific font that I need to recognize, but the model from this package doesn't have a good accuracy.
It could be possible use a custom model, from TensorFlow, Pytorch, etc.

**Comment by fbernaly** [comment]
No custom models are supported for text recognition.
You can read more about it in Google's documentation: https://developers.google.com/ml-kit/vision/text-recognition/v2
